features/testbot/basic.feature

```diff
@@ -251,3 +251,34 @@ Feature: Basic Routing
             | from | to | route |
             | d    | c  | de,ce,ce |
             | e    | d  | de,de    |
+
+    Scenario: Ambiguous edge weights - Use minimal edge weight
+        Given the node map
+            | a | b |
+
+        And the ways
+            | nodes | highway   |
+            | ab    | tertiary  |
+            | ab    | primary   |
+            | ab    | secondary |
+
+        When I route I should get
+            | from | to | route | time |
+            | a    | b  | ab,ab | 10s  |
+            | b    | a  | ab,ab | 10s  |
+
+    Scenario: Ambiguous edge names - Use maximal name id
+        Given the node map
+            | a | b | c |
+
+        And the ways
+            | nodes | highway | name |
+            | ab    | primary | 1    |
+            | ab    | primary |      |
+            | ab    | primary | 2    |
+            | bc    | primary | 3    |
+
+        When I route I should get
+            | from | to | route |
+            | a    | c  | 2,3,3 |
+            | c    | a  | 3,2,2 |
```

Comment:
I think that might be an option. Otherwise we end up with just another undefined behaviour.